tests/DevApps/daemon-app/Daemon-app/Daemon-app.csproj

```diff
@@ -6,7 +6,7 @@
     <RootNamespace>Daemon_app</RootNamespace>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
-    <LangVersion>10.0</LangVersion>
```

Comment:
The line shouldn't be needed

Reply:
It is required for this .csproj as it is not using the common tests/Directory.Build.props.

Reply:
and tests don't use LangVersion 11?

Reply:
Test do use 11 however, this csproj is not importing tests/Directory.Build.props at all.

Reply:
@FuPingFranco : that's not the way it works.
projects don't need to import the `Directory.Build.props` file. dotnet build/MSBuild will use the closest one (that is the one in the folder of the project, or the closest parent).

Directory.Build.props files themselves, if they want to override the behavior of the closest parent Directory.Build.props need to import it and override properties.

Reply:
however, in the case of the ASP.NET MVC projects, there is Directory.Build.Proj which does not include anything from it's parent, and therefore that's were we'd want the `<LangVersion>`:
[
](https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/DevApps/aspnet-mvc/Directory.Build.props)

Reply:
and in the case of DevApps, there is another one, which also only explicits the target frameworks.
https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/DevApps/Directory.Build.props
that's were we'd want the <LangVersion>:

Reply:
In the case of Devapps this works however, in asp.net seems to still be picking 7.3 for some reason which causes issues with nullables already enabled. 

<img width="452" alt="error_msg" src="https://github.com/AzureAD/microsoft-identity-web/assets/38921563/2db0e190-f59b-42bc-93f1-b23d1191e48f">